src/configureStore.ts

```diff
@@ -133,12 +135,8 @@ export function configureStore<S = any, A extends Action = AnyAction>(
   if (devTools) {
     finalCompose = composeWithDevTools({
       // Enable capture of stack traces for dispatched Redux actions
-
-      // @ts-ignore redux-devtools-extension doesn't have `trace` defined in
-      // its type definition file yet:
-      //
-      // https://github.com/zalmoxisus/redux-devtools-extension/pull/624
-      trace: !IS_PRODUCTION
+      trace: !IS_PRODUCTION,
+      ...typeof devTools === 'object' && devTools,
```

Comment:
Perhaps I can do this in a different way? I played with a couple different ways, this just seemed terse and retained behavior.